meta-mender-core/classes/mender-setup-image.inc

```diff
@@ -34,7 +34,7 @@ MENDER_BOOT_PART_MOUNT_LOCATION:mender-grub:mender-bios = "/boot/grub"
 EFI_PREFIX:mender-image = "${MENDER_BOOT_PART_MOUNT_LOCATION}"
 
 # Update fstab for Mender
-ROOTFS_POSTPROCESS_COMMAND:append = " mender_update_fstab_file;"
+ROOTFS_POSTPROCESS_COMMAND:append = " mender_update_fstab_file "
```

Comment:
Here, and below, why not then use the other append syntax?
```suggestion
ROOTFS_POSTPROCESS_COMMAND += "mender_update_fstab_file"
```

Reply:
Hi, i used the other append syntax here; 2d8a4c829f81364a1a5a2a73b88812d2bdc48bc0 
(the commit was squashed)

Reply:
Thanks :zap: 

But please let's be consistent, add it also to [the line below](https://github.com/mendersoftware/meta-mender/pull/2264/files#diff-10d8fb7e7b9d26931f191d2c25b334b55f264d07516ba8788c776ebde43a87d1R95).

Reply:
done (and squashed) here: aa8837d57b9112d0ca54edb6d20d4b1ec9322c8e